README.md

```diff
@@ -4,13 +4,31 @@
 
 Repository for students to experiment with in ITP's Open Source Studio Class.
 
-## More about this
+## Stuff about markdown
 
 I am demonstrating markdown.
 
-### Another subheading
+### Some basic markdown
 
 A list
 * thing 1
 * thing 2
 * thing 3
+
+### Extra markdown
+
+__Fun__ _with_ [markdown](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf)
+
+Wow! | Much | Organise
```

Comment:
Should we use International or American spelling?

Reply:
international!